.gitignore

```diff
@@ -11,3 +11,4 @@ generated_assists.adoc
 generated_features.adoc
 generated_diagnostic.adoc
 .DS_Store
+.glitterrc
```

Comment:
Let's remove this -- we are trying to keep .gitignore minimal.

Reply:
It's there for a reason - I added glitter config because that has my personal config, but I guess I can remove it.

Reply:
You can use a global gitignore. It is stored at `$XDG_CONFIG_HOME/git/ignore` on Linux. (The default of `$XDG_CONFIG_HOME` is `$HOME/.config`)

Reply:
I use it in other projects tho